x-pack/plugins/ml/public/ui_actions/constants.ts

```diff
@@ -6,3 +6,4 @@
  */
 
 export const CONTROLLED_BY_SWIM_LANE_FILTER = 'anomaly-swim-lane';
+export const CONTROLLED_BY_EXPLORER_FILTER = 'anomaly-explorer';
```

Comment:
This might need to change to `anomaly-chart`.

Reply:
Renamed here [`3c31e37` (#94396)](https://github.com/elastic/kibana/pull/94396/commits/3c31e37dacc795ad9e4b0b71d459616bda58b979)